ganttproject/src/biz/ganttproject/impex/csv/GanttCSVExport.java

```diff
@@ -153,7 +153,11 @@ public void save(SpreadsheetWriter writer) throws IOException {
   }
 
   private String i18n(String key) {
-    return GanttLanguage.getInstance().getText(key);
+    String result = GanttLanguage.getInstance().getText(key);
```

Comment:
I suggest using `InternationalizationKt.getRootLocalizer().formatText(key)` instead. It returns the key if there's no translation.